grapher/file_grapher.py

```diff
@@ -1,10 +1,10 @@
 from collections import namedtuple
 import os
 import re
+import sys
 
 import jedi
 
-
```

Comment:
This is as per [`PEP8`](https://www.python.org/dev/peps/pep-0008/#blank-lines) style guidelines. Which is the "official" Python style guide.

> Surround top-level function and class definitions with two blank lines.